docs/mkdocs/docs/api/macros/json_diagnostic_positions.md

```diff
@@ -6,17 +6,20 @@
 
 This macro enables position diagnostics for generated JSON objects.
 
-When enabled, two new properties: `start_pos()` and `end_pos()` are added to `nlohmann::json` objects and fields. `start_pos()` returns the start position of
+When enabled, two new properties: `start_pos()` and `end_pos()` are added to `nlohmann::basic_json` objects and fields. `start_pos()` returns the start position of
 that JSON object/field in the original string the object was parsed from. Likewise, `end_pos()` returns the end position of that JSON object/field in the
 original string the object was parsed from.
 
-For objects and arrays, the start and end positions represent the positions of the opening and closing braces or brackets, respectively. For fields, the start
-and end positions represent either the opening and closing quotes for that field's value or the first and character after last in the field's numerical or
-predefined true/false/null values.
+For objects and arrays, the start position represents the position of the opening brace or bracket. The end position is the character after the closing
+brace or bracket. For fields, the start position represents the opening quote or first character in field's numerical or predefined (true/false/null) value,
+the end position represents the character after the closing quote or the character after the last character in the field's numerical or predefined value.
 
-`start_pos()` and `end_pos()` are only set if the JSON object was parsed using `json::parse()`. For all other cases, `std::string::npos` will be returned.
+Given the above, `end_pos() - start_pos()` for an object or field provides the length of the string representation for that object or field, including the
```

Comment:
Just double checking if the back ticks should include the operation or just the operands. Dunno what the standard they use elsewhere is.

Reply:
They didn't seem to have an issue with the operator being in the backticks, I will leave it this way